manifests/c/c0re100/qBittorrent-Enhanced-Edition/4.5.4.10/c0re100.qBittorrent-Enhanced-Edition.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with WinGet Releaser v2 using Komac v1.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: c0re100.qBittorrent-Enhanced-Edition
+PackageVersion: 4.5.4.10
+Platform:
+- Windows.Desktop
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Protocols:
+- magnet
+FileExtensions:
+- torrent
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/c0re100/qBittorrent-Enhanced-Edition/releases/download/release-4.5.4.10/qbittorrent_enhanced_4.5.4.10_qt6_x64_setup.exe
+  InstallerSha256: D5D59180F389FDCF3185B4D62FBE97DEB3FB68C87E96A32A894DCA13B5D0C04E

```

Comment:
```suggestion
```

Reply:
Sorry, It looks like something is broken.
I confirmed x86 installer exists, but Releaser submit incorrect InstallerUrl for x86.
I will push changes later.

issue: https://github.com/c0re100/qBittorrent-Enhanced-Edition/issues/488